### ISSUE TITLE
Zero documentation on hundreds separator for currencies

### ISSUE THREAD

**Issue by wwwhatley** [uNmAnNeR/imaskjs#258]
Looking for a simple $2.50 ... or something similar--I've been unable to locate any documentation or examples of this implemented...

**Comment by sem4phor** [comment]
You can accomplish this easily with combining pattern and number mask.

https://imask.js.org/guide.html#number
https://imask.js.org/guide.html#pattern -> blocks section

```
{
      mask: '$ num ',
      blocks: {
        num: {
          mask: Number,
          scale: 2
        }
      }
    }
```

For trailing Currency-Sign I had to invoke the dynamic masks feature, too. But for your use case you are fine with above solution.

**Comment by uNmAnNeR** [comment]
closing this in favor of #170
